robot/web/web_test.go

```diff
@@ -863,13 +927,13 @@ func TestRawClientOperation(t *testing.T) {
 	var hdr metadata.MD
 	_, err = client.GetStatus(ctx, &robotpb.GetStatusRequest{}, grpc.Header(&hdr))
 	test.That(t, err, test.ShouldBeNil)
-	checkOpID(hdr)
+	checkOpID(hdr, true)
 
 	streamClient, err := client.StreamStatus(ctx, &robotpb.StreamStatusRequest{})
 	test.That(t, err, test.ShouldBeNil)
 	md, err := streamClient.Header()
 	test.That(t, err, test.ShouldBeNil)
-	checkOpID(md)
+	checkOpID(md, false)
```

Comment:
ah yes, please use my little thing in that case. i'd just copy it over since it's not a real resource we work with. it's okay to be filtering stream status. the test when i wrote it just looked for something that streamed :)